extensions/db/src/main/java/org/mule/extension/db/internal/operation/BulkOperations.java

```diff
@@ -62,8 +61,8 @@
    *         according to the order in which commands were added to the batch.
    * @throws SQLException if an error is produced
    */
-  public int[] bulkInsert(@ParameterGroup BulkQueryDefinition query,
-                          @XmlHints(allowReferences = false) List<Map<String, Object>> parameterValues,
+  public int[] bulkInsert(@XmlHints(allowReferences = false) List<Map<String, Object>> parameterValues,
```

Comment:
Makes sense, rather have an explicit @Placement annotation to leave things clear.